packaging/windows/OpenRA.nsi

```diff
@@ -140,13 +140,9 @@ Section "Game" GAME
 	File "${SRCDIR}\VERSION"
 	File "${SRCDIR}\AUTHORS"
 	File "${SRCDIR}\COPYING"
-	File "${SRCDIR}\README.html"
```

Comment:
That is quite radical, but I get it. Our README mostly tells people on how to compile the game which isn't interesting if you already downloaded the binaries.

Reply:
My main target was the changelog, the readme was collateral damage that lets us get rid of the markdown dependencies. I could live with adding this back if someone can make a case for it.